voice.js

```diff
@@ -0,0 +1,183 @@
+let personName = "";
+let toDoList = [];
+
+function greetWithName(commandArray) {
+  const userName = commandArray.slice(-1);
+  if (personName) {
+    console.log(`You already mention your name is : ${userName}`);
+    return;
```

Comment:
This `return` is not necessary. You chose to use either an `if ... else ...` statement or an `if` with early return, but not both😅


```js
// if ... else solution
if (personName) {
   console.log(`You already mention your name is : ${userName}`);
} else {
   personName = userName;
   console.log(`Nice to meet you,${userName}.`);
}

// early return solution
if (personName) {
   return `You already mention your name is : ${userName}`;
} 
   
personName = userName;
return `Nice to meet you,${userName}.`;
```

Reply:
I'll let you choose a solution and implement it in the other functions as well.